prototypes/ItemRepeaterPrototypes/ItemsRepeaterExperiments/Container/DifferentTemplateTests.xaml

```diff
@@ -0,0 +1,147 @@
+﻿<Page
+    x:Class="ItemsRepeaterExperiments.Container.DifferentTemplateTests"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:ItemsRepeaterExperiments.Container"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:muxc="using:Microsoft.UI.Xaml.Controls" xmlns:common="using:ItemsRepeaterExperiments.Common"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel>
+
+        <TextBlock Text="Standard template"/>
+        <muxc:ItemsRepeaterScrollHost Grid.Row="2">
+            <ScrollViewer HorizontalScrollBarVisibility="Auto" VerticalScrollBarVisibility="Auto">
+                <muxc:ItemsRepeater Grid.Row="2" ItemsSource="{x:Bind teams,Mode=OneWay}">
+
+                    <muxc:ItemsRepeater.Layout>
+                        <muxc:StackLayout Orientation="Horizontal"/>
+                    </muxc:ItemsRepeater.Layout>
+
+                    <muxc:ItemsRepeater.ItemTemplate>
+                        <DataTemplate x:DataType="common:Team">
+
+                            <!-- Handles managing visual states, such as pressing, handling clicks etc -->
+                            <local:HandleContainer>
+
+                                <!-- Inline UserControl -->
+                                <StackPanel x:Name="RootPanel" BorderBrush="Transparent" 
+                                    Padding="8"
+                                    BorderThickness="1" MinHeight="200" MinWidth="200">
+                                    <VisualStateManager.VisualStateGroups>
```

Comment:
why would these visual states not be part of HandleContainer ? Seems wasteful to have visual states on both the container and inline.

Reply:
Forgot to delete them, removed them now.